docs/io.cozy.procedures.md

```diff
@@ -0,0 +1,31 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Procedures doctype
+
+## `io.cozy.procedures`
+
+The `io.cozy.procedures` doctype represents the administrative procedures.
+
+- `templateId`: {string} the id of the `io.cozy.procedures.templates` the procedure is related to.
+- `data`: {object} the data about the procedure (_e.g_ amount, duration, _etc_).
+- `personalData`: {object} the personal data associated to the procedure (_e.g_ email, phone number, _etc_).
+- `relationships`: {object} links between documents
+  - `files`: {object} attachments
+    - `data`: {array} list of files
+      - `_id`: {string} id of the io.cozy.files document
+      - `_type`: {string} doctype "io.cozy.files"
+
+## `io.cozy.procedures.templates`
+
+The `io.cozy.procedures.templates` doctype represents the administrative procedures templates.
+
+- `name` {string} the procedure name (_e.g_ credit application).
+  <!-- - `data` {object} -->
+  <!-- - `personalData` {object} -->
+- `files` {array} the number and types of `io.cozy.files` required for this type of procedure.
+  - `label` {string}
+  - `class` {string}
+  - `subtype` {string}
+  - `count` {number}
+  - `rules` {array}
+  - `accept` {string}
```

Comment:
I think we can remove this template ATM. It's not clear enough on how we should do that since the rules are not yet specified

Reply:
I removed it, it is up to date in the paper and the .js file in the code will be a reference for this if we want to create a template doctype in the future